EasyPost.Tests/UserTest.cs

```diff
@@ -105,70 +128,47 @@ public async Task TestUpdate()
             Dictionary<string, object> userDict = new Dictionary<string, object>
             {
                 {
-                    "name", testName
+                    "name",
+                    testName
                 }
             };
-            await user.Update(userDict);
+            user = await user.Update(userDict);
 
-            Assert.IsInstanceOfType(user, typeof(User));
-            Assert.IsTrue(user.id.StartsWith("user_"));
-            Assert.AreEqual(testName, user.name);
+            Assert.IsType<User>(user);
+            Assert.StartsWith("user_", user.id);
+            Assert.Equal(testName, user.name);
         }
 
-        [TestMethod]
+        [Fact]
+        [CrudOperations.Delete]
         public async Task TestDelete()
         {
-            _vcr.SetUpTest("delete");
+            UseVCR("delete");
 
             User user = await CreateUser();
 
-            bool success = await user.Delete();
-            Assert.IsTrue(success);
+            await user.Delete();
 
-            _userId = null; // skip deletion cleanup
-        }
+            // TODO: Assert something
```

Comment:
Issue: Let's assert something now for the same reasons as the other delete functions.

Reply:
Nothing to assert, since the function does not return anything.

Reply:
Same comment as above ^, does C# not provide a "assert no error"? I prefer we be explicit over implicit.